app/models/host/managed.rb

```diff
@@ -291,7 +291,8 @@ def handle_realm
   def diskLayout
     @host = self
     template = disk.blank? ? ptable.layout : disk
-    pxe_render(template.tr("\r", ''))
+    template_name = disk.blank? ? ptable.name : 'Custom disk layout'
+    unattended_render(template.tr("\r", template_name))
```

Comment:
Yeah I filed this for discussion rather than for testing. I am unsure how will you like this change.